src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -370,20 +370,20 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
                 return 0;
             }
 
-            if (!_recievedFirstByte)
+            if (!_receivedFirstByte)
             {
                 const auto now = std::chrono::high_resolution_clock::now();
                 const std::chrono::duration<double> delta = now - _startTime;
 
 #pragma warning(suppress : 26477 26485 26494 26482 26446) // We don't control TraceLoggingWrite
                 TraceLoggingWrite(g_hTerminalConnectionProvider,
-                                  "RecievedFirstByte",
-                                  TraceLoggingDescription("An event emitted when the connection recieves the first byte"),
+                                  "ReceivedFirstByte",
```

Comment:
It's possible this is an API of sorts.

Reply:
Nnnnn. @amit-msft, does this mess up reporting if we correct the typo? Or can you glue the two together on the back-end?